.travis.yml

```diff
@@ -62,6 +62,13 @@ matrix:
       - CMAKE_EXTRA_FLAGS="-C ../host-configs/rocm.cmake -DCMAKE_CXX_COMPILER=/opt/rocm/bin/hcc -DENABLE_HCC=On -DENABLE_OPENMP=Off -DENABLE_CUDA=Off -DENABLE_BENCHMARKS=Off -DENABLE_WARNINGS_AS_ERRORS=Off"
       - DO_TEST=no
 
+   - compiler: hip
+     env:
+       - COMPILER=g++
+       - IMD=rocm
+       - CMAKE_EXTRA_FLAGS="-C ../host-configs/hip.cmake"
+       - DO_TEST=no
```

Comment:
What is the plan for running CI for this?

Reply:
Travis, but I'm removing it for now since the Travis build didn't seem to be running. I will add it back in a task branch once this gets merged.